src/Integrations/Integrations/Roadrunner/RoadrunnerIntegration.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace DDTrace\Integrations\Roadrunner;
+
+use DDTrace\SpanData;
+use DDTrace\Integrations\Integration;
+use DDTrace\Type;
+use DDTrace\Util\Normalizer;
+
+/**
+ * Roadrunner integration
+ */
+class RoadrunnerIntegration extends Integration
+{
+    const NAME = 'roadrunner';
+
+    /**
+     * @return string The integration name.
+     */
+    public function getName()
+    {
+        return self::NAME;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function requiresExplicitTraceAnalyticsEnabling()
+    {
+        return false;
+    }
+
+    /**
+     * @return int
+     */
+    public function init()
+    {
+        if (!self::shouldLoad(self::NAME)) {
+            return Integration::NOT_LOADED;
+        }
+
+        $integration = $this;
+
+        ini_set("datadog.trace.auto_flush_enabled", 1);
+        ini_set("datadog.trace.generate_root_span", 0);
+
+        \DDTrace\hook_method('Spiral\RoadRunner\Http\PSR7Worker', 'waitRequest', [
+            'prehook' => function () use (&$activeSpan) {
+                if ($activeSpan) {
+                    \DDTrace\close_spans_until($activeSpan);
+                    \DDTrace\close_span();
+                }
+            },
+            'posthook' => function ($psr, $scope, $args, $retval, $exception) use (&$activeSpan, $integration) {
+                if (!$retval && !$exception) {
+                    return; // shutdown
+                }
+
+                /** @var \Psr\Http\Message\ServerRequestInterface $retval */
+                $activeSpan = \DDTrace\start_trace_span();
+                $activeSpan->service = \ddtrace_config_app_name('roadrunner');
+                $activeSpan->name = "web.request";
```

Comment:
@labbati This is a new integration. How confident are you in `http.server`? Should we set it to `web.request` for now?

Reply:
I would just keep it consistent, until we change them all.